npm/webpack-preprocessor/__snapshots__/compilation.spec.js

```diff
@@ -8,9 +8,9 @@ Webpack Compilation Error
 Module build failed (from ./node_modules/babel-loader/lib/index.js):
 SyntaxError: <path>/_test-output/syntax_error_spec.js: Unexpected token (1:18)
 
-[0m[31m[1m>[22m[39m[90m 1 |[39m describe([32m'fail'[39m[33m,[39m [33m-[39m[33m>[39m)[0m
-[0m [90m   |[39m                   [31m[1m^[22m[39m[0m
-[0m [90m 2 |[39m[0m
+[0m[31m[1m>[22m[39m[90m 1 |[39m describe([32m'fail'[39m[33m,[39m [33m-[39m[33m>[39m)
```

Comment:
needed to update the snapshot. Likely due to the babel updates 
<img width="824" alt="Screenshot 2024-06-05 at 11 56 38 PM" src="https://github.com/cypress-io/cypress/assets/3980464/c71c4850-7e0b-467d-8eaa-e91ffb92b587">